client/Android/Studio/freeRDPCore/src/main/res/xml/screen_settings.xml

```diff
@@ -37,5 +37,11 @@
             freerdp:bounds_default="480"
             freerdp:bounds_max="65536"
             freerdp:bounds_min="480" />
+       <com.freerdp.freerdpcore.utils.IntListPreference
+            android:entries="@array/scale_array"
+            android:entryValues="@array/scale_values_array"
+            android:key="bookmark.scale"
+            android:summary="Scaling Percentage"
```

Comment:
* no hard coded strings, move to `strings.xml`.
* Summary is not really helpful, this is `HighDPI` support, so remote scaling, not local `/smart-sizing` like